menas/ui/components/app.controller.js

```diff
@@ -92,7 +94,19 @@ sap.ui.define([
     },
 
     onRunsPress: function (oEv) {
-      this._eventBus.publish("runs", "list");
+      this._eventBus.publish("runs", "list-grouped-name");
+      this._app.toMaster(this.createId("runsDatasetNamePage"));
+    },
+
+    onRunsDatasetNamePress: function (channel, event, data) {
+      this._eventBus.publish("runs", "list-grouped-version", data);
+      this._app.toMaster(this.createId("runsDatasetVersionPage"));
+    },
+
+    onRunsDatasetVersionPress: function (channel, event, dataset) {
+      console.log(dataset)
```

Comment:
Is this `console.log` intentional? If so, it is missing a `;`

Reply:
Hah, nope, missed it